functionaltests/8_15_test.go

```diff
@@ -148,5 +148,10 @@ func TestUpgrade_8_15_4_to_8_16_0(t *testing.T) {
 
 	res, err := ecc.GetESErrorLogs(ctx)
 	require.NoError(t, err)
-	assert.Zero(t, res.Hits.Total.Value)
+	if !assert.Zero(t, res.Hits.Total.Value, "expected no error logs, but found some") {
+		t.Log("found error logs:")
+		for _, h := range res.Hits.Hits {
+			t.Log(h.Source_)
```

Comment:
I think we need to stringify the `[]byte` 

```suggestion
			t.Log(string(h.Source_))
```